RecoLocalCalo/HcalRecProducers/plugins/alpaka/Mahi.h

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
```suggestion
#ifndef RecoLocalCalo_HcalRecProducers_plugins_alpaka_Mahi_h
#define RecoLocalCalo_HcalRecProducers_plugins_alpaka_Mahi_h

```

Reply:
(and the closing `#endif` at the end)

Reply:
Done in 715105876f8999402d3e30f2fa94579275479978